dao-utils/src/parts/error.rs

```diff
@@ -44,6 +62,14 @@ impl From<u16> for Error {
             1004 => Error::TotalSupplyOverflow,
             1005 => Error::ValueNotAvailable,
             1006 => Error::ActivationTimeInPast,
```

Comment:
You missed NoSuchMethod

Reply:
On purpose, having u16 we don't have the name that we can pass to `NoSuchMethod`. This conversion does not make sense to me.